plugin/rocks.lua

```diff
@@ -6,6 +6,25 @@ local nio = require("nio")
 local adapter = require("rocks.adapter")
 local config = require("rocks.config.internal")
 
+-- Initialize the luarocks loader
+if config.enable_luarocks_loader then
+    local sc = vim.system({ config.luarocks_binary, "which", "luarocks.loader" }):wait()
+    local luarocks_path = sc.stdout and sc.stdout:match("(%S+)loader.lua")
+    if luarocks_path then
+        package.path = package.path .. ";" .. luarocks_path .. "?.lua"
+        vim.uv.os_setenv("LUAROCKS_CONFIG", config.luarocks_config)
```

Comment:
```suggestion
        vim.env.LUAROCKS_CONFIG = config.luarocks_config
```

nit: would this not be the simpler approach, or does it have some drawbacks I'm not aware of? :)